src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts

```diff
@@ -237,4 +255,61 @@ export class TakenWerkvoorraadComponent
     this.selection.clear();
     this.dataSource.filtersChanged();
   }
+
+  private handleAssigmentOrReleasementWorkflow<T>(
```

Comment:
I would opt for `handleAssignOrReleaseWorkflow`, reason being the word Releasement is not used much anymore, and for both Assignment and Releasement we can use the base verbs, Assign and Release, to convey the same meaning.

Reply:
And I notice a typo in `Assigment`, should be `Assignment`...

Reply:
Agreed. In the backend we also use the word `release` to good to keep in sync.

ps: I never heard of the term `releasement` up till now. ;-)

Reply:
done :-)